packages/components/src/CodeEditor.tsx

```diff
@@ -25,7 +24,7 @@ export let CodeEditor: FC<CodeEditorProps> = ({
       value={value}
       mode="golang"
       theme="github"
-      width={width + "px"}
+      width={"100%"}
```

Comment:
This seems to make some really long text boxes:
![image](https://user-images.githubusercontent.com/13807753/162108450-f26229f3-bb84-474e-92cd-c333214a3510.png)

Reply:
It would be nice if the code editor changed width along with the graph

Reply:
I think the graph should change too, but that's more work, so will get onto later.

Typically, I think that having components accept pixel widths is not a good idea. Often you want the inner components to scale to a fraction of the parent components, and you also don't have the pixel width of the outer component, so trying to have good pixel widths for the inner components is a pain.

Reply:
Fair enough, we can get around to that later